azure-monitor-opentelemetry-distro/azure/monitor/opentelemetry/distro/_status_logger.py

```diff
@@ -0,0 +1,52 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+import socket
+from json import dumps
+from os import getpid, makedirs
+from os.path import exists, join
+
+from azure.monitor.opentelemetry.distro._constants import (
+    _CUSTOMER_IKEY,
+    _EXTENSION_VERSION,
+    _IS_DIAGNOSTICS_ENABLED,
+    _get_log_path
+)
+from azure.monitor.opentelemetry.distro._version import VERSION
+
+_MACHINE_NAME = socket.gethostname()
+_STATUS_LOG_PATH = _get_log_path(status_log_path=True)
+
+
+def _get_status_json(agent_initialized_successfully, pid, reason=None):
+    status_json = {
+        "AgentInitializedSuccessfully": agent_initialized_successfully,
+        "AppType": "python",
+        "MachineName": _MACHINE_NAME,
```

Comment:
Curious, `MachineName` represents an ip address?

Reply:
That would be `gethostbyname` I believe. But `gethostname` gives the host name of the device. For me it's LAPTOP-jvoss. For Helen's app service box, it was RD0003FFBAABF0.

Reply:
We have something similar in the [exporter](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/_utils.py#L123) which also returns similar to LAPTOP-jvoss. Maybe we shuold use `platform.node()` to be consistent.

Reply:
Can do. Produces the same result for me.